src/assets/scss/Pages/_pdf-list.scss

```diff
@@ -17,51 +17,58 @@
 }
 
 #tab_PDF {
-
   #gfpdf_list_form {
 
     .gfpdf-spinner {
       width: 20px;
     }
 
-    /* Modify the Gravity Forms 2.6 shortcode button version to fit in our PDF list table. */
-    td.column-shortcode {
-      .gform-button {
-        width: 100%;
-        text-align: center;
-        justify-content: center;
-      }
-    }
-
-    .btn-shortcode + .gpdf-fallback-input {
-      display: none;
-    }
+    table {
+      tbody {
+        & > :nth-child(2n+1) {
```

Comment:
No need to overcomplicate this. `td:nth-child(odd)` will give you the same result and be much easier to understand.